src/tabs/tab.jsx

```diff
@@ -13,7 +13,10 @@ const Tab = React.createClass({
 
   propTypes: {
     onTouchTap: React.PropTypes.func,
-    label: React.PropTypes.string,
+    label: React.PropTypes.oneOfType([
```

Comment:
I'm wondering if `React.PropTypes.node` would be better here.
According to the doc of facebook
> Anything that can be rendered: numbers, strings, elements or an array